projects/kinde-angular/src/lib/cookie-manager.ts

```diff
@@ -5,8 +5,13 @@ export class CookieManager {
     return Cookies.get(name);
   }
 
-  static setCookie(name: string, value: string): void {
-    Cookies.set(name, value, { path: '' });
+  static setCookie(name: string, value: string, options = {}): void {
+    Cookies.set(name, value, {
+      path: '',
+      secure: true,
+      sameSite: 'lax',
```

Comment:
Could we add `httpOnly` here also?

Reply:
Seems like es-cookie doesn't support that flag, TS will throw an error and checking the source code it won't [do anything with it](https://github.com/theodorejb/es-cookie/blob/master/src/es-cookie.ts#L19). Alternative solutions would be if we try to extend the interface with `declare module`. Will check for an alternative to es-cookie..

Reply:
Checked it a bit more indepth, httpOnly is not available client side. Having that set true means JavaScript can't access it. See [rfc6265 - 4.1.2.6 the HttpOnly Attribute](https://datatracker.ietf.org/doc/html/rfc6265#section-4.1.2.6)